src/main/java/com/vinted/ab/TestConfiguration.java

```diff
@@ -67,4 +66,56 @@ public TestConfiguration(String salt, int bucketCount, List<AbTest> abTests) {
 
         return result;
     }
+
+    /**
+     * @param identifier user identifier
+     * @return List of all (running + ended) tests assigned to user
+     */
+    public List<AbTest> getAllTests(String identifier) {
+        BigInteger bucketId = getAssignedBucketId(identifier);
+
+        List<AbTest> tests = new ArrayList<>();
+        for (AbTest test : abTests) {
+            if ((test.isAllBuckets() || test.getBuckets().contains(bucketId.intValue()))) {
```

Comment:
I honestly have no idea, I took this part from `getRunningTests`. @slapsys any ideas? :)